code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm

```diff
@@ -750,3 +793,505 @@ var/bloodstone_backup = 0
+		qdel(src)
+		return
+
+	var/image/I_circle = image(icon, src, "dance_platform_empty")
+	I_circle.plane = relative_plane(ABOVE_TURF_PLANE)
+	I_circle.layer = ABOVE_TILE_LAYER
+	I_circle.appearance_flags |= RESET_COLOR
+	overlays += I_circle
+	transform *= 0.3
+	source = runespell
+	processing_objects += src
+	idle_pulse()
+
+	spawn(5)
+		animate(src, alpha = 255, transform = matrix(), time = 4)
+
+/obj/effect/cult_ritual/dance_platform/Destroy()
+	if (dancer && prisoner)
+		dancer.SetStunned(-4)
+	dancer = null
+	source = null
+	dance_manager = null
+	processing_objects -= src
+	..()
+
+/obj/effect/cult_ritual/dance_platform/process()
+	if (dancer && prisoner)
+		dancer.SetStunned(4)
+
+	if (dancer)
+		if (dancer.loc != loc)
+			dancer = null
+			source.lost_dancer()
+			prisoner = FALSE
+			overlays.len = 0
+			var/image/I_circle = image(icon, src, "dance_platform_empty")
+			I_circle.plane = relative_plane(ABOVE_TURF_PLANE)
+			I_circle.layer = ABOVE_TILE_LAYER
+			I_circle.appearance_flags |= RESET_COLOR
+			overlays += I_circle
+		else
+			source.dance_increment(dancer)
+	else
+		for (var/mob/living/carbon/C in loc)
+			if (valid_dancer(C))
+				break
+
+/obj/effect/cult_ritual/dance_platform/Crossed(var/atom/movable/mover)
+	valid_dancer(mover)
+
+/obj/effect/cult_ritual/dance_platform/proc/valid_dancer(var/atom/movable/mover)
+	if (!dancer && !moving)
+		if (iscarbon(mover))
+			var/mob/living/carbon/C = mover
+			if (C.mind && !C.isDead())
+				if (iscultist(C))
+					dancer = C
+					overlays.len = 0
+					var/image/I_circle = image(icon, src, "dance_platform_full")
+					I_circle.plane = relative_plane(ABOVE_TURF_PLANE)
+					I_circle.layer = ABOVE_TILE_LAYER
+					I_circle.appearance_flags |= RESET_COLOR
+					var/image/I_markings = image(icon, src,"dance_platform_markings")
+					I_markings.plane = relative_plane(OBJ_PLANE)
+					I_markings.layer = BELOW_TABLE_LAYER
+					overlays += I_circle
+					overlays += I_markings
+					source.dancer_check(C)
+					return TRUE
+				else
+					if (istype(C.handcuffed,/obj/item/weapon/handcuffs/cult))
+						dancer = C
+						prisoner = TRUE
+						dancer.SetStunned(4)
+						overlays.len = 0
+						var/image/I_circle = image(icon, src, "dance_platform_full")
+						I_circle.plane = relative_plane(ABOVE_TURF_PLANE)
+						I_circle.layer = ABOVE_TILE_LAYER
+						I_circle.appearance_flags |= RESET_COLOR
+						var/image/I_markings = image(icon, src,"dance_platform_markings")
+						I_markings.plane = relative_plane(OBJ_PLANE)
+						I_markings.layer = BELOW_TABLE_LAYER
+						overlays += I_circle
+						overlays += I_markings
+						var/image/I = image('icons/obj/cult.dmi', src, "dance_prisoner")
+						I.plane = relative_plane(ABOVE_LIGHTING_PLANE)
+						I.layer = NARSIE_GLOW
+						overlays += I
+						to_chat(C, "<span class='danger'>Dark tentacles emerge from the rune and trap your legs in place. You'll need to remove those cuffs or get some help if you are to escape this circle.</span>")
```

Comment:
"You'll need to remove those cuffs or..." -> "You'll need to resist out of the tentacles or..." to help players understand how to break out.

Reply:
I'm changing the wording a bit so it's clearer that resisting is the thing to do here.